skimage/draw/_random_shapes.py

```diff
@@ -257,10 +284,15 @@ def random_shapes(image_shape,
     intensity_range : {tuple of tuples of uint8, tuple of uint8}, optional
         The range of values to sample pixel values from. For grayscale images
         the format is (min, max). For multichannel - ((min, max),) if the
-        ranges are equal across the channels, and ((min_0, max_0), ... (min_N, max_N))
-        if they differ. As the function supports generation of uint8 arrays only,
-        the maximum range is (0, 255). If None, set to (0, 254) for each
-        channel reserving color of intensity = 255 for background.
+        ranges are equal across the channels, and
+        ((min_0, max_0), ... (min_N, max_N)) if they differ. As the function
+        supports generation of uint8 arrays only, the maximum range is
+        (0, 255). If None, set to (0, 255) for each channel reserving color of
```

Comment:
`reserving color of intensity = 255 for background.` <- this should probably be moreved. The same info is provided 2 lines below.